source/dem/dem.cc

```diff
@@ -499,6 +506,56 @@ DEMSolver<dim>::finish_simulation()
             }
         }
     }
+
```

Comment:
I think it is better to move all these new functions in the dem.cc to separate files.

Reply:
Like i create a a new dem function that i insert in the finish_simulation function. Or i insert it after the finish_simulation in the solve function?

Reply:
I still don't catch it, which new functions? And your comment point to dem.cc file so these function are already in ...

Reply:
no look at input_parameter_inspection function for example. It is defined in a separate file (input_parameter_inspection.cc and .h) and not inside the dem.cc

Reply:
Has this been addressed?